project/TypelevelScalafixPlugin.scala

```diff
@@ -0,0 +1,111 @@
+import sbt._, Keys._
+
+import de.heikoseeberger.sbtheader.HeaderPlugin.autoImport._
+import org.typelevel.sbt._
+import scalafix.sbt._
+import ScalafixPlugin.autoImport._
+import ScalafixTestkitPlugin.autoImport._
+import TypelevelSettingsPlugin.autoImport._
+
+object TypelevelScalafixPlugin extends AutoPlugin {
```

Comment:
Does this name collide with TypelevelScalafixPlugin from `sbt-typelevel`. Come to think of it, should we extract this into sbt-typelevel?

Reply:
Woops, I suppose it does 😂 

But yes, I do think we should eventually upstream this to sbt-typelevel or even sbt-scalafix if they'll take it. I need to think harder about how to disentangle some stuff (like how this currently depends on the header, settings, and no-publish plugins).